test_average (__main__.TestExercice) ... ok
test_to_celsius (__main__.TestExercice) ... FAIL
test_to_degrees (__main__.TestExercice) ... FAIL
test_to_farenheit (__main__.TestExercice) ... ok
test_to_radians (__main__.TestExercice) ... FAIL

======================================================================
FAIL: test_to_celsius (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 71, in test_to_celsius
    self.assertListEqual(
AssertionError: Lists differ: [-17.77777777777778, -31.666666666666668, 232.7777777777778] != [-17.77777777777778, -31.666666666666664, 232.77777777777777]

First differing element 1:
-31.666666666666668
-31.666666666666664

- [-17.77777777777778, -31.666666666666668, 232.7777777777778]
?                                        ^                  ^

+ [-17.77777777777778, -31.666666666666664, 232.77777777777777]
?                                        ^                  ^^
 : Mauvais calcul de farenheit -> celsius

======================================================================
FAIL: test_to_degrees (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 59, in test_to_degrees
    self.assertListEqual(
AssertionError: Lists differ: [(114.59155902616465, 0.03333333333333333, [216 chars]112)] != [(79.0980174562859, 5.881047377154118, 29.6[197 chars]998)]

First differing element 0:
(114.59155902616465, 0.03333333333333333, 0.0005555555555555556)
(79.0980174562859, 5.881047377154118, 29.612494192724625)

- [(114.59155902616465, 0.03333333333333333, 0.0005555555555555556),
-  (-229.1831180523293, -0.06666666666666667, -0.0011111111111111111),
-  (286.4788975654116, 0.08333333333333333, 0.001388888888888889),
-  (234.33973820850667, 0.06816666666666667, 0.0011361111111111112)]
+ [(79.0980174562859, 5.881047377154118, 29.612494192724625),
+  (-278.1960349125718, 48.23790524569176, 0.7750116145507491),
+  (257.74504364071475, -15.297381557114704, 44.031235481811564),
+  (213.95544569810457, -2.673258113725865, 23.057550624127998)] : Mauvais calcul de radians -> degres

======================================================================
FAIL: test_to_radians (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 40, in test_to_radians
    self.assertListEqual(
AssertionError: Lists differ: [-0.0[13 chars]503, 0.17598251810595045, 6.283490739798685, 6.997344948274799] != [-0.0[13 chars]503, 0.17598251810595048, 6.283490739798685, 6.9973449482748]

First differing element 1:
0.17598251810595045
0.17598251810595048

+ [-0.08668468618238503, 0.17598251810595048, 6.283490739798685, 6.9973449482748]
- [-0.08668468618238503,
-  0.17598251810595045,
-  6.283490739798685,
-  6.997344948274799] : Mauvais calcul de degres -> radians

----------------------------------------------------------------------
Ran 5 tests in 0.002s

FAILED (failures=3)
